### DISCUSSION TITLE
is there a way to translate datenumber to string?

### DISCUSSION THREAD

**Discussion by cuihee** [handsontable/hyperformula#1463] [API Questions]
I want to get string like "2024-11-20 16:50:00.000"
I have datenumber like 45616.777
is there a function to do that?

**Comment by AMBudnik** [comment]
Hey @cuihee 

I'm not sure if that would meet your requirements, but there's a method called `numberToDateTime`.

It returns an object with the following syntax example

```
{year: 2020, month: 1, day: 15, hours: 2, minutes: 24, seconds: 0}
```

Reference: https://hyperformula.handsontable.com/api/classes/hyperformula.html#numbertodatetime

going from `{year: 2020, month: 1, day: 15, hours: 2, minutes: 24, seconds: 0}` to  `2024-11-20 16:50:00.000` is a matter of formatting and concatenation. 

Would that work for you?

**Reply by cuihee** [reply]
Useful! Can fulfill my functional requirements
